core/pom.xml

```diff
@@ -96,6 +96,12 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-auth</artifactId>
+            <exclusions>
```

Comment:
Maybe we can manage this exclusion in the `DependencyManagement` block of parent pom too?

Reply:
> Maybe we can manage this exclusion in the `DependencyManagement` block of parent pom too?

Thanks for your remind, done

> Maybe we can manage this exclusion in the `DependencyManagement` block of parent pom too?

Thanks for your remind, done